docs/guides/ce-resource-management.rst

```diff
@@ -0,0 +1,67 @@
+Compute Engine Resource Management
+=======================
+
+Workflows submitted to run on Compute Engine can specify the computational resources they require, enabling precise management of resources and costs.
+
+.. note::
+    Resource management is supported only when executing on Compute Engine runtimes. Tasks and workflows defined with these parameters may still be executed on other runtimes, however the resource specification will be ignored.
+
+Setting Task Resources
+----------------------
+
+Required hardware resources are configured on a per-task basis by setting the ``resources`` field of the task decorator:
+
+.. code-block::
+   :caption: task resource request example
+
+    @sdk.task(
+        resources=sdk.Resources(cpu="100m", memory="1Gi", disk="10Gi", gpu="1")
+    )
+    def my_task():
+        ...
+
+``resources`` expects a ``sdk.Resources()`` object that specifies some or all of:
+* number of cores (``cpu``)
+* amount of RAM (``memory``)
+* disk space (``disk``)
+* number of GPU computing units (``gpu``)
+
+The example above specifies a task that requires 100 milicores (or 0.1 cores), 1 Gb of RAM, 10 Gb of disk space, and access to a GPU.
+
+Setting Workflow Resources
+--------------------------
+
+Resources can also be configured at the workflow definition level using the same syntax as with tasks:
+
+.. code-block::
+    :caption: workflow resource request example
+
+    @sdk.workflow(
+        resources=sdk.Resources(cpu="100m", memory="1Gi", disk="10Gi", gpu="1")
```

Comment:
Can you explain what the different suffixes mean?
What's "100m" CPUs, how does it relate to a single core. What about "Gi" vs "Mi" vs "G" vs "M" for memory.

Reply:
[9b2fd18](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/96/commits/9b2fd18710b2b04cb2a8e0b085ae691a68d72837)